app/src/main/java/com/github/se/travelpouch/MainActivity.kt

```diff
@@ -42,12 +44,29 @@ fun TravelPouchApp() {
   val navigationActions = NavigationActions(navController)
   val listTravelViewModel: ListTravelViewModel = viewModel(factory = ListTravelViewModel.Factory)
 
-  NavHost(navController = navController, startDestination = Route.OVERVIEW) {
+  NavHost(navController = navController, startDestination = Route.AUTH) {
     navigation(
-        startDestination = Screen.ADD_TRAVEL,
-        route = Route.OVERVIEW,
+        startDestination = Screen.AUTH,
+        route = Route.AUTH,
     ) {
+      composable(Screen.AUTH) { SignInScreen(navigationActions) }
+    }
+
+    navigation(
+        startDestination = Screen.GREETING,
+        route = Route.GREETING,
+    ) {
+      composable(Screen.GREETING) { Greeting() }
```

Comment:
the navigation should be modified to have only AUTH and ADD_TRAVEL

Reply:
This is temporary and will be replaced by the overview screen (see issue #13) in issue #53.